omod/src/main/webapp/pages/labs/labWorkflow.gsp

```diff
@@ -16,123 +17,94 @@
-    const dateTimeFormat = '${pihAppsConfig.getGlobalProperty("uiframework.formatter.JSdateAndTimeFormat", "DD-MMM-YYYY HH:mm")}';
-    const primaryIdentifierType = '${pihAppsConfig.primaryIdentifierType?.uuid ?: ""}';
-    const conceptUtils = new PihAppsConceptUtils(jq);
 
-    <% orderStatuses.each{ s -> %>
-        window.translations['${s}'] = '${ui.message("pihapps.orderStatus." + s)}';
-    <% } %>
-    <% fulfillerStatuses.each{ s -> %>
-        window.translations['${s}'] = '${ui.message("pihapps.fulfillerStatus." + s)}';
-    <% } %>
-
-    const getEmrId = function(order) {
-        const sortedIdentifiers = order.patient.identifiers.sort(function(a, b) {
-            const aPrimary = a.identifierType.uuid = primaryIdentifierType;
-            const bPrimary = b.identifierType.uuid = primaryIdentifierType;
-            if (aPrimary && !bPrimary) {
-                return -1;
-            }
-            if (!aPrimary && bPrimary) {
-                return 1;
-            }
-            if (a.preferred && !b.preferred) {
-                return -1;
-            }
-            if (!a.preferred && b.preferred) {
-                return 1;
-            }
-            const aCreated = new Date(a.identifierType.auditInfo.dateCreated).getTime();
-            const bCreated = new Date(b.identifierType.auditInfo.dateCreated).getTime();
-            return aCreated - bCreated;
-        });
-        return sortedIdentifiers[0].identifier;
-    }
-
-    const getPatientName = function(order) {
-        return order.patient.person.display;
-    }
-
-    const getOrderDate = function(order) {
-        const m = moment(order.dateActivated);
-        if ( m.hour() === 0 && m.minute() === 0 && m.second() === 0 && m.millisecond() === 0) {
-            return m.format(dateFormat);
-        }
-        return m.format(dateTimeFormat);
-    }
-
-    const getOrderNumber = function(order) {
-        return order.orderNumber;
-    }
-
-    const getAccessionNumber = function(order) {
-        return order.accessionNumber;
-    }
-
-    const getOrderStatus = function(order) {
-        if (order.dateStopped) {
-            return window.translations['STOPPED'];
-        }
-        if (order.autoExpireDate && moment(order.autoExpireDate).isBefore(new Date())) {
-            return window.translations['EXPIRED'];
-        }
-        return window.translations['ACTIVE'];
-    }
-
-    const getFulfillerStatus = function(order) {
-        return order.fulfillerStatus ? window.translations[order.fulfillerStatus] : "${ ui.message('pihapps.none') }";
-    }
-
-    const getLabTest = function(order) {
-        const urgency = order.urgency === 'STAT' ? '<i class="fas fa-fw fa-exclamation" style="color: red;"></i>' : '';
-        return urgency + conceptUtils.getConceptShortName(order.concept, window.sessionContext?.locale);
-    }
+    const pagingDataTable = new PagingDataTable(jq);
 
     jq(document).ready(function() {
```

Comment:
I had to rework this page to put a lot more logic inside of the block that runs after the pihappsConfig is retrieved from the server.  I also re-worked it for the new status filter we discussed.